networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -29,9 +29,11 @@ type GameTemplate @model{
   phaseOneTime: Int!
   phaseTwoTime: Int!
   imageUrl: String
-  questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions")
+  questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions") 
+  numQuestionTemplates: Int! @default(value: "0")
   createdAt: AWSDateTime
```

Comment:
field to track number of `questionTemplates` in a game per our call last week

Reply:
sounds good. I've replaced `numQuestionTemplates` with `questionTemplatesCount` and `numGameTemplates` with `GameTemplatesCount`